src/Cassandra.IntegrationTests/TestClusterManagement/Simulacron/Cluster.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+
+namespace Cassandra.IntegrationTests.TestClusterManagement.Simulacron
+{
+    public class Cluster : Base
```

Comment:
You could use `SimulacronCluster` and `SimulacronBase` to avoid conflicts and dealing with `using` statements.
Just an opinion, its your call.